com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Project.java

```diff
@@ -1800,8 +1804,10 @@ private List<TaskResult> runTasks(IProgress monitor) throws IOException {
                     }
                 }
                 TimeProfiler.stop();
-
-                boolean shouldRun = (!allOutputExists || !allSigsEquals) && !completedTasks.contains(task);
+                // game.project is always last task
+                boolean isLastTask = completedTasks.size() + 1 == buildTasks.size();
+                boolean shouldRun = !completedTasks.contains(task);
```

Comment:
I didn't want to change it as part of my fix, but we don't process completed tasks and this is required condition, shouldn't we exit earlier?
![CleanShot 2024-10-05 at 16 53 59@2x](https://github.com/user-attachments/assets/4871733f-0b60-4f7b-a254-75c15e45265c)